mvpa2/algorithms/hyperalignment.py

```diff
@@ -430,7 +437,12 @@ def _level1(self, datasets, commonspace, ref_ds, mappers, residuals):
             # to make a batch update after processing all 1st-level datasets
             # to an identical 1st-level common space
             # TODO: make just a function so we dont' waste space
-            commonspace = params.combiner1(ds_, commonspace)
+            if params.level1_equal_weight:
```

Comment:
Is the TODO still applicable?

Reply:
who knows... probably still is (for "inplace" zscoring, right @swaroopgj?), but it is unrelated to this PR